pkg/detector/preemption.go

```diff
@@ -263,25 +253,21 @@ func (d *ResourceDetector) fetchResourceTemplate(rs policyv1alpha1.ResourceSelec
 // and put the PropagationPolicy in the queue to trigger preemption.
 func (d *ResourceDetector) HandleDeprioritizedPropagationPolicy(oldPolicy policyv1alpha1.PropagationPolicy, newPolicy policyv1alpha1.PropagationPolicy) {
 	klog.Infof("PropagationPolicy(%s/%s) priority changed from %d to %d", newPolicy.GetNamespace(), newPolicy.GetName(), *oldPolicy.Spec.Priority, *newPolicy.Spec.Priority)
-	policies, err := d.propagationPolicyLister.ByNamespace(newPolicy.GetNamespace()).List(labels.Everything())
+	policyList := &policyv1alpha1.PropagationPolicyList{}
+	err := d.Client.List(context.TODO(), policyList, client.InNamespace(newPolicy.GetNamespace()))
```

Comment:
Why not we set `UnsafeDisableDeepCopy` here?

Reply:
> Why not we set UnsafeDisableDeepCopy here?

It should be set, it should be missed